src/pages/concerts/Concerts.js

```diff
@@ -1,73 +1,20 @@
+// Hooks
+import { useCollection } from '../../hooks/useCollection';
 // Material components
 import Grid from '@mui/material/Grid';
 // Components
 import ConcertCard from '../../components/concerts/ConcertCard/ConcertCard';
 
-const concerts = [
-  {
-    id: 1,
-    title: 'Varsad Bhinjave',
-    image: 'https://media.timeout.com/images/101206597/750/422/image.jpg',
-    venue: 'VNSGU, Surat',
-    time: '9 PM, 10th August, 2022',
-    artists: [
-      'Prashant Patel',
-      'Nishant Chavda',
-      'Hriday Sur',
-      'Sunil Revar',
-      'Parth Khalasi',
-      'Jinesh Shah',
-      'Prem Dave',
-      'Hetaxi Trivedi',
-      'Nidhi Adhvaryu',
-      'Prachi Shukla',
-      'Nishi Shukla',
-    ],
-  },
-  {
-    id: 2,
-    title: 'Sur Shabda Ne Sathvare',
-    image:
-      'https://www.israel21c.org/wp-content/uploads/2022/03/main-pic-2.jpg',
-    venue: 'VNSGU, Surat',
-    time: '9 PM, 10th August, 2022',
-    artists: [
-      'Prashant Patel',
-      'Nishant Chavda',
-      'Hriday Sur',
-      'Sunil Revar',
-      'Parth Khalasi',
-      'Jinesh Shah',
-      'Prem Dave',
-      'Hetaxi Trivedi',
-      'Nidhi Adhvaryu',
-      'Prachi Shukla',
-      'Nishi Shukla',
-    ],
-  },
-  {
-    id: 3,
-    title: 'Aavi Shake To Aav',
-    image: 'https://media.timeout.com/images/101206597/750/422/image.jpg',
-    venue: 'VNSGU, Surat',
-    time: '9 PM, 10th August, 2022',
-    artists: [
-      'Prashant Patel',
-      'Nishant Chavda',
-      'Hriday Sur',
-      'Sunil Revar',
-      'Parth Khalasi',
-      'Jinesh Shah',
-      'Prem Dave',
-      'Hetaxi Trivedi',
-      'Nidhi Adhvaryu',
-      'Prachi Shukla',
-      'Nishi Shukla',
-    ],
-  },
-];
-
 const Concerts = () => {
+  const { documents: concerts, isPending, error } = useCollection('concerts');
```

Comment:
Can we please create a constant CollectionName class with 'concerts', 'lyrics' etc etc.. & Use this 'concerts' name in all the component from this class?

Reply:
Done